load_testing/api.sh

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
+
+set -e
+
+source ./.env.sh || (echo 'Please create a `.env.sh` file based on the `.env.sh.template` file.' > /dev/stderr && false)
+
+host=${1:-http://localhost:8080}
```

Comment:
Why is `http://localhost:8080` used as a default here, but `http://172.17.0.1:8000` in the justfile?

Reply:
It should probably just require that a host name is passed instead of duplicating the default, right?

To address the issue you had where you have to change the host to `host.docker.internal` anyway, I'm going to add platform detection to the justfile and determine the correct "localhost" default to use from there and remove this one from the script.